openacademy/wizards/openacademy_wizard_views.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="wizard_view_form" model="ir.ui.view">
+        <field name="name">openacademy.wizard.view.form</field>
+        <field name="model">openacademy.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Wizard">
+                <sheet>
+                    <group name="wizards" string="Wizard">
+                        <field name="session_ids" />
+                        <field name="partner_ids"/>
```

Comment:
Use also here the widget `many2many_tags`

Reply:
But, how could **multiple** attendees be added if not through a tree view?
In the Vauxoo tutorial a tree view is used for add the attendees -> https://vauxoo.thinkific.com/courses/take/odoo-backend/lessons/1966754-ejercicio-asistentes

Reply:
I really don't understand you, sorry. The following code is the one that is present in this PR, and it works:
```python
    @api.model
    def default_get(self, fields_list):
        result = super().default_get(fields_list)
        model = self.env.context.get("active_model")
        if model == "openacademy.session":
            session_ids = self.env.context.get("active_ids")
            sessions = self.env.get(model).browse(session_ids)
            result.update({"session_ids": sessions})
        return result
```

Reply:
I mean:

```
@api.model
def default_get(self, fields_list):
    result = super().default_get(fields_list)
    active_model = self.env.context.get("active_model")
    active_ids = self.env.context.get("active_ids")
    if active_model == "openacademy.session" and active_ids:
        result["session_ids"] = [(6, 0, active_ids)]
    return result
```

Reply:
Okay, I ask something, when the `active_model = "openacademy.session"`, doesn't this determine that the `active_ids` is already set? In the worst case the `active_ids` would be an empty list.

The notation `[(6, 0, active_ids)]` doesn't look very clear to me, we haven't seen it in the tutorial.

Reply:
An fix or improvement could be this:
```python
    @api.model
    def default_get(self, fields_list):
        result = super().default_get(fields_list)
        model = self.env.context.get("active_model")
        session_ids = self.env.context.get("active_ids")
        if model == "openacademy.session" and session_ids:
            sessions = self.env.get(model).browse(session_ids)
            result.update({"session_ids": sessions})
        return result
```

Reply:
> In the worst case the active_ids would be an empty list.

In this case, yes. But it's a good practice to write code as robust as possible. In the future, you could implement wizards that are called not only from the user interface, but also by code.

> The notation [(6, 0, active_ids)] doesn't look very clear to me

In deed, it isn't. That's why in v15 it was replaced by [the Command class](https://www.odoo.com/documentation/15.0/developer/reference/backend/orm.html#odoo.fields.Command). Unfortunately, the old syntax is still all around the code (people don't even know it yet), so you will need to get familiarized with ithe old syntax, you can read [the doc of v14](https://www.odoo.com/documentation/14.0/developer/reference/addons/orm.html##create-update). For this specific case, it would be:

> (6, 0, ids)
> 
>     replaces all existing records in the set by the ids list

> A fix or improvement could be this:

In your code, you're browsing Ids, which creates a recordset, but only IDs are used in the end. Besides, it's generally a better idea to return Ids when dealing with dictionaries, as they work in both code and views (managed by frontend). Maybe not as necessary on newer versions, though.